sdk/keyvault/keyvault-admin/recordings/node/keyvaultbackupclient/recording_beginbackup_then_beginselectiverestore.js

```diff
@@ -300,120 +282,264 @@ nock('https://eastus2.keyvault_name.managedhsm-int.azure-int.net:443', {"encoded
   'x-ms-keyvault-network-info',
   'addr=108.226.109.105',
   'x-ms-server-latency',
-  '526',
+  '541',
   'content-security-policy',
   'default-src \'self\'' ]);
 
-nock('https://eastus2.keyvault_name.managedhsm-int.azure-int.net:443', {"encodedQueryParams":true})
-  .put('/keys/beginBackupthenbeginSelectiveRestore/restore', {"sasTokenParameters":{"storageResourceUri":"https://chrissprim.blob.core.windows.net/backup","token":"?sv=2019-12-12&ss=bfqt&srt=sco&sp=rwdlacupx&se=2022-07-14T05:56:52Z&st=2020-09-03T21:56:52Z&spr=https,http&sig=Rzts1hBG%2BgqTCTZo%2BX%2FZgJWJ5Ao16ueN%2F1BB4Dg8%2FLo%3D"},"folder":"https://chrissprim.blob.core.windows.net/backup"})
+nock('https://eastus2.keyvault_name.managedhsm.azure.net:443', {"encodedQueryParams":true})
+  .put('/keys/rsa-1/restore', {"sasTokenParameters":{"storageResourceUri":"https://uri.blob.core.windows.net/backup","token":"blob_storage_sas_token"},"folder":"mhsm-keyvault_name-2020091100085704"})
```

Comment:
This is a existing key we're using for testing in Java too.

Reply:
Using an existing key simplifies the tests considerably. Over time we can expand and make more in depth tests.